### ISSUE TITLE
Primary Button fails accessibility contrast standards on dark backgrounds

### ISSUE THREAD

**Issue by petewoodhouse-rg** [carbon-design-system/carbon-contribution#11]
**Name of the component:**
Primary Button (Carbon V2 dark theme)

**Explanation of change**
Color changes - currently the Primary Button fails accessibility contrast standards on dark backgrounds (UI components should have at least 3:1 contrast)

Change = Primary Button color to Blue 50 with Gray 100 text

**The research supporting the change**

> **UI components**
> Visual information used to indicate states and boundaries of UI components must have a contrast ratio of 3:1 against adjacent colors. A UI component is defined as “a part of the content that is perceived by users as a single control for a distinct function.” This includes form elements.

https://next.carbondesignsystem.com/guidelines/accessibility/color

**Screenshots of the proposed design**

<img width="846" alt="screen shot 2019-02-25 at 21 30 48" src="https://user-images.githubusercontent.com/9609126/53371026-23f17a00-3947-11e9-92c7-8d0b33c04378.png">

<img width="845" alt="screen shot 2019-02-25 at 21 31 36" src="https://user-images.githubusercontent.com/9609126/53371060-35d31d00-3947-11e9-98f8-29aff4d78e1c.png">

**Comment by asudoh** [comment]
Moved to: IBM/carbon-components#1914
